app/models/voucher.rb

```diff
@@ -167,6 +167,7 @@ def inverted_rows(options = {})
     rows.reject!(&:canceled?) unless options[:canceled]
     rows.map do |old|
       vr = old.clone
```

Comment:
Isn't `clone` potentially dangerous? I think `dup` is the proper way to do this.

(Ping @Koronen?)